example/main.cpp

```diff
@@ -1,131 +0,0 @@
-//========================================================================================
```

Comment:
I thought we deleted this file a while ago. Why is it showing up in the diff now?

Reply:
https://github.com/lanl/parthenon/blob/master/example/main.cpp

Reply:
huh ok I could have sworn we deleted this a while ago. My bad.